lib/src/widgets/reactive_form_field.dart

```diff
@@ -143,8 +143,12 @@ class ReactiveFormFieldState<ModelDataType, ViewDataType>
   /// See [ControlValueAccessor].
   @protected
   @visibleForTesting
-  ControlValueAccessor<dynamic, dynamic> selectValueAccessor() {
-    return DefaultValueAccessor<ModelDataType>();
+  ControlValueAccessor<ModelDataType, ViewDataType> selectValueAccessor() {
+    assert(
+      ModelDataType == ViewDataType,
+      "Must provide a valueAccessor for field $widget as ModelDataType and ViewDataType don't match.",
```

Comment:
I don't understand very well this assert. Could you explain me? or make a test to see why this is useful?

Reply:
I understand now, Good Job

Reply:
But please add a test for this new assert.

Reply:
I will try to get this done tomorrow.

Reply:
Added tests